autowire/implementation.py

```diff
@@ -78,3 +78,21 @@ def manager(*args, **kwargs):
 
         super().__init__(manager, arg_resources, kwarg_resources)
         self.fn = fn
+
+
+class PlainValueImplementation(ContextManagerImplementation[R]):
```

Comment:
Maybe `ConstantImplementation` would be a better name

Reply:
Thanks. Fixed.